jsapp/js/components/modalForms/translationSettings.es6

```diff
@@ -315,18 +333,19 @@ export class TranslationSettings extends React.Component {
                       }
                     </bem.FormView__iconButton>
 
+                    <bem.FormView__iconButton
+                      data-index={i}
+                      data-string={this.state.translations[i]}
+                      onClick={this.launchTranslationTableModal}
+                      disabled={this.state.isUpdatingDefaultLanguage}
+                      data-tip={t('Update translations')}
+                      className='right-tooltip'
+                    >
+                      <i className='k-icon-globe-alt' />
+                    </bem.FormView__iconButton>
+
                     {i !== 0 &&
                       <React.Fragment>
```

Comment:
You can remove this `<React.Fragment>` too

Reply:
I think I could've cleaned up a bit better, they must've been leftovers from every test I've made. Sorry bout that!